### PR TITLE
[CS-2830] Update payment request expanded sheet, fix input flickering issue on Android

### PR DESCRIPTION
**Pull request by topwebtek7**
### Description

Moved requestPaymentRequest screen from ExpandedAssetSheet to new cardstack route with new sheet component, and also updated sheet component to have fixed Header, Footer. also did some UI tweaks for different device/screen sizes.

- [x] Completes #CS-2830

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

Screen Shot 2022-01-11 at 3 23 22 AM
Screen Shot 2022-01-11 at 3 22 51 AM

Screen Shot 2022-01-11 at 3 22 51 AM

### CODE REVIEW

File: cardstack/src/screens/PaymentRequest/RequestPaymentConfirmation.tsx [hunk 1]
```diff
@@ -218,29 +218,33 @@ export const RequestPaymentConfirmation = ({
...
+    style={
+      Device.isIOS
+        ? shadow.buildAsObject(0, -1, 2, 'rgba(0, 0, 0, 0.25)', 1)
+        : {
+            borderTopWidth: 0.5,
+            borderTopColor: 'rgba(0, 0, 0, 0.25)',
+            shadowColor: 'rgba(0, 0, 0, 0.25)',
+            elevation: 3,
+          }
+    }
```
Comment (danibonilha) [inline cardstack/src/screens/PaymentRequest/RequestPaymentConfirmation.tsx]: Can we extract this to an style const outside the component ?  If you get any typing error, you can just type the style as `StyleProp<ViewStyle>`

```tsx
const footerStyle: StyleProp<ViewStyle> = ... 
```

File: cardstack/src/components/Sheet/Sheet.tsx [hunk 2]
```diff
@@ -141,6 +145,7 @@ const Sheet = ({
         <CenteredContainer paddingVertical={4}>
           {!hideHandle && <SheetHandle />}
         </CenteredContainer>
+        {Header}
         <ScrollView
```
Comment (danibonilha) [inline cardstack/src/components/Sheet/Sheet.tsx:149]: nit: would make sense to use StickyHeader from ScrollView in this case ?

File: cardstack/src/screens/PaymentRequest/PaymentRequestExpandedSheet.tsx [hunk 3]
```diff
@@ -88,20 +98,16 @@ const PaymentRequestExpandedState = (props: { asset: MerchantSafeType }) => {
...
-      bottomInset={editMode ? 50 : 110}
-      hasKeyboard={editMode}
-      height="100%"
-      renderFooter={() =>
-        editMode ? <EditFooter /> : <RequestPaymentConfirmationFooter />
-      }
-      renderHeader={() => (
+    <Sheet
+      isFullScreen
+      scrollEnabled
```
Comment (danibonilha) [inline cardstack/src/screens/PaymentRequest/PaymentRequestExpandedSheet.tsx]: Maybe we should disable the scroll if we are at the editMode, bc on iOS we get bouncy behavior on the input.

**Review by danibonilha — APPROVED** [review]
Looking good, just small suggestions

File: cardstack/src/screens/PaymentRequest/PaymentRequestExpandedSheet.tsx (hunk 3 above)
Comment (topwebtek7) [reply cardstack/src/screens/PaymentRequest/PaymentRequestExpandedSheet.tsx]: good point! 👍🏼

File: cardstack/src/components/Sheet/Sheet.tsx (hunk 2 above)
Comment (topwebtek7) [reply cardstack/src/components/Sheet/Sheet.tsx:149]: hmm I've tried but didn't work with StickyHeader for some reason.. gonna merge this pr and will look at it more later

**Pull request merged**